app/views/annotations/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+      </label>
+      &nbsp
+      <%= t('admin.remark.preview') %>
+    </div>
+
+    <br>
+
+    <!--- COLOR SELECTION -->
+    <b><%= t('admin.annotation.color') %></b>
+    <div id="annotation-color-picker">
+      <% for i in 1..15 do %>
+        <%= f.radio_button :color,
+                           annotation_color(i),
+                           id: "annotation_color#{i}" %>
+        <%= f.label :color,
+                    "annotation_color#{i}",
+                    for: "annotation_color#{i}" do %>
+            <span class="annotation_color<%=i%>"></span>
+          <% end %>
+      <% end %>
+    </div>
+    <!--- SIMPLE COLOR PICKER
+    <b>Color: &nbsp</b>
+    <%= f.color_field :color %> -->
+
+    <br>
+
+    <!--- CATEGORY RADIO BUTTON -->
+    <b><%= t('admin.annotation.category') %></b>
+    <div class="form-row ml-2", id="category-radios">
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :note,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.note'),
+                    value: :note,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :content,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.content'),
+                    value: :content,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :mistake,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.mistake'),
+                    value: :mistake,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :presentation,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.presentation'),
+                    value: :presentation,
+                    class: 'form-check-label' %>
+      </div>
+    </div>
+
+    <br>
+
+    <div id="specific">
+      <!--- SPECIFIC CONTENT, DEPENDING ON THE SELECTED CATEGORY, IS INSERTED HERE -->
+    </div>
+
+    <br>
+
+    <!--- VISIBLE FOR TEACHER? - CHECKBOX -->
+    <% annotations_status = Medium.find_by(id: @medium_id).get_annotations_status %>
+    <% if annotations_status == 1 %>
+      <%= f.label :visible_for_teacher, class: "checkbox inline" do %>
+        <span, title="<%= t('admin.annotation.visible_for_teacher_tooltip') %>"><b>
+          <%= t('admin.annotation.visible_for_teacher') %> &nbsp
+        </b></span>
+        <%= f.check_box :visible_for_teacher %>
```

Comment:
Checkbox not at the same height as label:

<img width="415" alt="image" src="https://github.com/MaMpf-HD/mampf/assets/37160523/c5a79ad6-d757-4584-8bb1-2ca07d7a7749">